zipkin-lens/webpack.prod.config.js

```diff
@@ -66,6 +60,14 @@ module.exports = {
           minimize: true,
         },
       },
+      {
+        test: /webfonts\/.*\.(svg|woff|woff2|ttf|eot)$/,
+        loader: "file-loader",
```

Comment:
This can be replaced with `url-loader` if `url:` base64 inlined fonts are preferred. I prefer my fonts in separate files, but up to you.